ui/src/components/UserMenu/UserMenuButton.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component, Fragment } from 'react';
+import classNames from 'classnames';
+import { object } from 'prop-types';
+import { withApollo } from 'react-apollo';
+import Avatar from '@material-ui/core/Avatar';
+import { withStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import AccountCircleIcon from 'mdi-react/AccountCircleIcon';
+import Button from '../Button';
+import SignInDialog from '../SignInDialog';
+import { withAuth } from '../../utils/Auth';
+
+@withStyles(theme => ({
+  icon: {
+    fill: theme.palette.common.white,
```

Comment:
Is this having any effect? I think we can remove it.

Reply:
yes, we can remove this.